app/src/main/java/com/ajdi/yassin/popularmovies/ui/moviedetails/trailers/TrailerViewHolder.java

```diff
@@ -35,7 +35,7 @@ public TrailerViewHolder(@NonNull ItemTrailerBinding binding, Context context) {
     public void bindTo(final Trailer trailer) {
         String thumbnail =
                 "https://img.youtube.com/vi/" + trailer.getKey() + "/hqdefault.jpg";
-        GlideApp.with(context)
+        Glide.with(context)
```

Comment:
Does this improve anything? Please revert this change.

Reply:
There is no `GlideApp` class defined in the project, thus it gives compile-time and build errors. Replacing it with the default Glide implementation fixed the problem.

Reply:
Did you successfully build the project before making any changes?

Reply:
Yes I did that, that's how I encountered this error.

Reply:
`GlideApp` is a part of Glide's [Generated API](https://bumptech.github.io/glide/doc/generatedapi.html), please read the doc for more understanding.

Reply:
Never mind, I fixed it back. Just needed to switch the annotation processor for Glide to `kapt` .

Reply:
Sweet. Can you also ensure to revert this change in other files too e.g. `BindingAdapters.java`.

Reply:
Done.